java-manta-client-unshaded/src/test/java/com/joyent/manta/util/CipherClonerTest.java

```diff
@@ -96,7 +97,7 @@ private void canCloneCipher(final SupportedCipherDetails cipherDetails) throws E
         final byte[] inputData = RandomUtils.nextBytes(cipherDetails.getBlockSizeInBytes() * 3);
 
         // notice we are specifically calling getBouncyCastleCipher()
```

Comment:
Comment no longer is true.

Reply:
@dekobon we should remove this comment, right ?